File: PHP/src/Jakopec/PartijaDvaIgraca.php
<?php


namespace Jakopec;

use Doctrine\ORM\Mapping\Entity;

/**
 * @Entity
 */
class PartijaDvaIgraca extends Partija
{
    public function __toString(): string
    {

        $rezultat = $this->getRezultat();
   return 'Partija DVA IGRAČA, igra gotova: ' . ($this->isIgraGotova() ? 'DA': 'NE') . ', ' . $this->getIgraci()[0] . ': ' .
       $rezultat->getPrvi() .
       ' | ' . $this->getIgraci()[1] . ': ' . $rezultat->getDrugi();
    }
}

File: Java/src/main/java/jakopec/model/PartijaDvaIgraca.java
package jakopec.model;

import jakarta.persistence.DiscriminatorValue;
import jakarta.persistence.Entity;
import jakarta.persistence.MappedSuperclass;
import jakarta.persistence.Table;
import org.hibernate.annotations.Polymorphism;
import org.hibernate.annotations.PolymorphismType;

@Entity
@DiscriminatorValue("dvaIgraca")
@Polymorphism(type = PolymorphismType.EXPLICIT)
public class PartijaDvaIgraca extends Partija {


    @Override
    public String toString() {
        Rezulat rezultat = getRezultat();
        return "Partija DVA IGRAČA, igra gotova: " + (isIgraGotova() ? "DA" : "NE") + ", " + getIgraci().get(0) + ": " +
                rezultat.getPrvi() +
                " | "+ getIgraci().get(1) + ": " + rezultat.getDrugi();
    }

}
